### ISSUE TITLE
Add ability to sign in with Google, save animations to a server, & share animations via URL

### ISSUE THREAD

**Issue by alexjlockwood** [alexjlockwood/ShapeShifter#137]
Obviously super long term goal, but it'd be pretty bad ass IMO.

**Comment by dalisoft** [comment]
Also you can use your domain to server with direct link like this
link.shapeshifter.design/a9g72
or
Dashboard.shapeshifter.design/user/app/id3

I am worked for Php and dashboard while frontend was been jquery ui 6years ago and since fail freelance i was stopped working php. If you create project with nodejs (use free cloudno.de) and Angular for both front/back end i will help

**Comment by alexjlockwood** [comment]
Good idea, thanks for the suggestion and for the offer! Will let you know if/when I ever start working on this!
